### PR TITLE
Add proper FAQ page and TTS instructions

### PR DESCRIPTION
**Pull request by biodranik**
@gpesquero @strump @rtsisyk

1. I removed other items to simplify merging the structure.
2. Also removed operating systems terms temporarily, maybe it would be better to create a separate taxonomy for them.

Please test and review this branch and let me know if we can proceed with publishing to production.

### CODE REVIEW

**Comment by gpesquero** [comment]
> Please test and review this branch and let me know if we can proceed with publishing to production.

It looks Ok to me...

File: README.md [hunk 1]
```diff
@@ -25,6 +25,48 @@ Every merge into the _master_ branch deploys changes into the production at http
...
+```yaml
+taxonomies:
+  faq: ["iOS"]
+```
+
+Zola will add your question to specific F.A.Q. sub-page.
+
+**Limitation**: List of taxonomies at `/faq/` page is always alphabetical. So 'Android' is always the first, 'Bookmarks' is the second, 'General' is the third and so on.
+
+**Limitation**: You can't sort questions at any F.A.Q. sub-page. So questions at `/faq/general` are always sorted by filename. We can create additional extra field like `extra.order` and sort questions by this field. But reordering would be hard.
```
Comment (strump) [inline README.md:65]: Actually we do use `extra.order` field for sorting topics.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,48 @@ Every merge into the _master_ branch deploys changes into the production at http
...
+After that Zola gets all values for `faq` taxonomy: `[General, Android, iOS, Map, ...]`. And generates pages:
+
+* For key `/faq/` with the list of values (see `templates/faq/list.html`)
+* For each value `/faq/general`, `/faq/android`, etc. with the list of questions (see `templates/faq/single.html`)
+
+If you want to add new question then create .md file with header:
+
+```yaml
+taxonomies:
+  faq: ["iOS"]
```
Comment (strump) [inline README.md:58]:
```suggestion
taxonomies:
  faq: ["iOS"]
extra:
  order: 40
```

File: README.md [hunk 3]
```diff
@@ -25,6 +25,48 @@ Every merge into the _master_ branch deploys changes into the production at http
...
+```
+
+Zola will add your question to specific F.A.Q. sub-page.
+
```
Comment (strump) [inline README.md:62]: Let's add:

> If you need to translate the FAQ to a new language please add next lines to `config.toml`:

```
[languages.XX]
taxonomies = [
  {name = "faq", feed = false},
]
[languages.XX.translations]
faq-menu-title = "{Translation of 'F.A.Q.' to a new language}"
```

**Comment by biodranik** [comment]
@gpesquero @strump @rtsisyk thanks for the review!

@borntojesus the FAQ structure is merged. Now we can proceed with the content in a similar way.

**Pull request merged**